ngraph/core/src/op/or.cpp

```diff
@@ -78,13 +81,7 @@ bool op::v1::LogicalOr::has_evaluate() const
     NGRAPH_OP_SCOPE(v1_LogicalOr_has_evaluate);
```

Comment:
Please add inputs/outputs tensor check to the evaluate method like here:
https://github.com/openvinotoolkit/openvino/blob/37ff06e5273826d3bf65902ae69494caf6d76f06/ngraph/core/src/op/and.cpp#L72-L78

Reply:
Added just as in the given example